data/json/monsters/zed_worm.json

```diff
@@ -0,0 +1,243 @@
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 95,
+    "speed": 80,
+    "material": [ "flesh" ],
+    "symbol": "Z",
+    "color": "red",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 5,
+    "melee_dice": 2,
+    "melee_dice_sides": 4,
+    "melee_damage": [ { "damage_type": "cut", "amount": 0 } ],
+    "dodge": 1,
+    "armor_elec": 1,
+    "bleed_rate": 50,
+    "vision_day": 30,
+    "vision_night": 3,
+    "weakpoint_sets": [ "wps_humanoid_body" ],
+    "families": [ "prof_intro_biology", "prof_wp_zombie" ],
+    "harvest": "zombie",
+    "special_attacks": [ { "type": "bite", "cooldown": 5 }, [ "scratch", 20 ], [ "tapeworm", 20 ] ],
+    "death_function": { "effect": { "id": "death_tapeworm", "hit_self": true } },
+    "death_drops": "mon_zombie_hulk_death_drops",
+    "zombify_into": "mon_zombie_rot",
+    "burn_into": "mon_zombie_scorched",
+    "fungalize_into": "mon_zombie_fungus",
+    "upgrades": { "half_life": 30, "into": "mon_zombie_worm_brute" },
+    "flags": [
+      "SEES",
+      "HEARS",
+      "SMELLS",
+      "STUMBLES",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "POISON",
+      "GRABS",
+      "NO_BREATHE",
+      "PUSH_MON",
+      "FILTHY",
+      "RANGED_ATTACKER"
+    ]
+  },
+  {
+    "id": "mon_zombie_worm_brute",
+    "type": "MONSTER",
+    "name": { "str": "spindly carrier" },
+    "description": "A zombie with a hunched, ape-like posture supported by surprisingly thin arms.  On the back itself resides an unnaturally large tapeworm, snaking around and twisting its hooked head.  Several smaller worms can be seen throughout the wounds on the body, the most developed ones wrapping themselves around the arms.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 200,
+    "speed": 80,
+    "material": [ "flesh" ],
+    "symbol": "Z",
+    "color": "red",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 5,
+    "melee_dice": 4,
+    "melee_dice_sides": 5,
+    "melee_damage": [ { "damage_type": "cut", "amount": 0 } ],
+    "dodge": 2,
+    "armor_bash": 3,
+    "armor_cut": 3,
+    "armor_elec": 1,
+    "bleed_rate": 0,
+    "vision_night": 5,
+    "weakpoint_sets": [ "wps_humanoid_body" ],
+    "families": [ "prof_intro_biology", "prof_wp_zombie" ],
+    "harvest": "zombie",
+    "special_attacks": [
+      [ "RANGED_PULL", 75 ],
+      { "type": "bite", "cooldown": 5 },
+      [ "scratch", 20 ],
+      {
+        "id": "tapeworm",
+        "damage_max_instance": [ { "damage_type": "stab", "amount": 14, "armor_penetration": 4 } ],
+        "range": 7
+      }
+    ],
+    "death_function": { "effect": { "id": "death_tapeworm", "hit_self": true } },
+    "death_drops": "mon_zombie_hulk_death_drops",
+    "zombify_into": "mon_zombie_grappler",
```

Comment:
Maybe add a monster group so 90% of the time it revives as a grappler zombie but the remaining 10% the stars align so a tapeworm can grow anew.

Reply:
The tapeworms that were to grow would use the corpse as the nutrients to grow, though, so the corpse would just be left in too sorry of a state to revive at that point. The idea is kind of cool I'm just not convinced how plausible that'd be.